messaging/Controllers/BundlesController.cs

```diff
@@ -128,7 +128,21 @@ public ActionResult PostIncomingMessageItem(string jurisdictionId, [FromBody] ob
             catch (Exception ex)
             {
                 _logger.LogDebug($"An exception occurred while parsing the incoming message: {ex}");
-                return BadRequest();
+                return BadRequest("Failed to parse message. Please verify that it is consistent with the current IG.");

```

Comment:
Suggest that instead of "IG" it say "Vital Records Messaging FHIR Implementation Guide"